.github/labeler.yml

```
@@ -158,3 +158,9 @@ HTML:
 JavaScript:
   - changed-files:
       - any-glob-to-any-file: '**/*.js'
+
+# test suite
+testsuite:
```

```suggestion
tests:
```
To be more generic than only testsuite, like when using pytest